source/features/bugs-tab.tsx

```diff
@@ -178,6 +178,9 @@ async function init(): Promise<void | false> {
 	}
 
 	await updateBugsTagHighlighting();
+	return () => {
+		console.trace('bugs-tab unloaded')
+	}
```

Comment:
This is a demo `deinit` handler to ensure that the feature is being unloaded when navigating away.